scripts/battlefields/Temenos/temenos_western_tower.lua

```diff
@@ -449,7 +458,7 @@ content.loot =
         },
     },
 
-    [7] =
+    [ID.TEMENOS_WESTERN_TOWER.npc.LOOT_CRATE] =
```

Comment:
I rested jmcmorris suggestion and it worked.

This change here is the only change needed

Reply:
👍